src/doom/ap_notif.c

```diff
@@ -65,7 +65,7 @@ cached_icon_t* get_cached_icon(patch_t* patch)
 
 
     // Scale down raw into cached icon (I inverted src and dst, too lazy to change)
-    int max_size = max(patch->width, patch->height);
+    int max_size = MAX(patch->width, patch->height);
```

Comment:
C min/max functions are not defined on linux. Instead, use MAX/MIN macro. These are defined in some header lib, but they are also defined in crispy.h so no need to add an include.